frontend/puppeteer/package-lock.json

```diff
@@ -1,8 +1,488 @@
 {
     "name": "puppeteer-debug-tool-for-mafia",
```

Comment:
Is this puppeteer file necessary to the issue? Was it just created by the npm install instruction, and do we want it to be added to the repo?

Reply:
I can't pass the husky lint check of commit if the import is missing, so an updated file is necessary.
But the file is not necessary for the issue